cumulusci/cli/service.py

```diff
@@ -35,7 +34,7 @@ def service_list(runtime, plain, print_json):
 
     console = Console()
     if print_json:
-        console.print(json.dumps(services))
+        console.print_json(data=services)
```

Comment:
Unrelated bug I found while adjusting test_service.py -- the old way could break lines in the middle of a string, so that the output wasn't parsable as JSON